core/components/atoms/avatarGroup/__stories__/variants/Size.story.jsx

```diff
@@ -2,29 +2,19 @@ import * as React from 'react';
 import { AvatarGroup, Text } from '@/index';
 import { list } from '../AvatarList';
 
-export const size = () => {
+export const Size = () => {
   return (
     <div className="d-flex">
-      <div className="d-flex flex-column">
+      <div className="flex-column mr-9 ">
         <Text weight="strong">Regular</Text>
-        <div className="d-flex">
-          <div className="mr-9 mt-4">
-            <AvatarGroup size="regular" list={list.slice(0, 4)} />
-          </div>
-          <div className="mr-9 bg-secondary-lightest p-4 ">
-            <AvatarGroup size="regular" borderColor="var(--secondary-lightest)" list={list.slice(0, 4)} />
-          </div>
+        <div className="mt-4">
+          <AvatarGroup size="regular" list={list.slice(0, 4)} />
         </div>
       </div>
-      <div className="d-flex flex-column">
+      <div className="flex-column">
         <Text weight="strong">Tiny</Text>
-        <div className="d-flex">
-          <div className="mr-9 mt-4">
-            <AvatarGroup size="tiny" list={list.slice(0, 4)} />
-          </div>
-          <div className="mr-9 bg-secondary-lightest p-3 mt-3 ">
-            <AvatarGroup size="tiny" borderColor="var(--secondary-lightest)" list={list.slice(0, 4)} />
-          </div>
+        <div className="mt-4">
+          <AvatarGroup size="tiny" list={list.slice(0, 4)} />
```

Comment:
Instead of wrapping it inside extra div please pass className `mt-4` directly to the AvatarGroup component